pettingzoo/butterfly/cooperative_pong/cooperative_pong.py

```diff
@@ -338,7 +341,7 @@ def render(self, mode="human"):
         return self.env.render(mode)
 
     def step(self, action):
-        if self.dones[self.agent_selection]:
+        if self.terminations[self.agent_selection]:
```

Comment:
this needs to be `if self.terminations[self.agent_selection] or self.truncations[self.agent_selection]` I think.